packages/frontend-web/src/test/notificationChecks.ts

```diff
@@ -26,80 +26,120 @@ import {
   checkUser,
 } from './objectChecks';
 
-export const globalUpdate = {
+export const globalUpdate: any = {
```

Comment:
You are mixing state (looks like some of `IGlobalSummary`) and actions on it here, but not doing it in a class. This makes things hard to type-check and to test. I'd recommend pulling out the data, giving it a type, and then having the functions affect the data. Or optionally having it as a class. This way you can also write tests, and understand what's happening easier.

Reply:
I'm really trying to achieve namespaces on the cheap.  But it is a pretty terrible way of doing it.  I'll try packing it in a class.

Reply:
Fixed: converted to a class.